examples/siro_sandbox/sandbox_app.py

```diff
@@ -750,21 +733,60 @@ def _parse_debug_third_person(args, framebuffer_size):
                 )
                 exit()
 
+            # avoid camera sensors for GUI-controlled agents
```

Comment:
One day soon, we should organize all this config-customization logic and other logic that runs at startup before `SandboxDriver.__init__`. E.g. put it into a class with readable helper functions. For now, this is fine.